src/__tests__/pdfParser.test.ts

```
@@ -407,3 +407,709 @@ describe('PDFParserService', () => {
+      
+      // Modifying returned array shouldn't affect future calls
+      formats1.push('fake-format');
+      const formats3 = PDFParserService.getSupportedFormats();
+      expect(formats3).not.toContain('fake-format');
+    });
+
+    it('estimateProcessingTime handles edge cases with file size', () => {
+      expect(PDFParserService.estimateProcessingTime(createFile(0))).toBe(5000);
+      expect(PDFParserService.estimateProcessingTime(createFile(-1000))).toBe(5000); // Negative size
+      expect(PDFParserService.estimateProcessingTime(createFile(Infinity))).toBe(60000);
+      expect(PDFParserService.estimateProcessingTime(createFile(NaN))).toBe(5000);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)